pyproject.toml

```diff
@@ -0,0 +1,92 @@
+[project]
+name = "molgrapher"
+version = "1.0.0"
+description = "Molgrapher"
+# TODO transform more old metadata over into here
```

Comment:
TODO add missing metadata from setup.py (if there is interest in getting this merged)